vault-pki-manager/vault-pki-manager.sh

```diff
@@ -14,10 +14,14 @@ mkdir "${output_dir}"
 if [[ -f /etc/tls/ca.crt ]] && [[ -n "${VAULT_CLIENT_NAMESPACES}" ]]; then
     echo "Updating CA in client namespaces"
     for n in ${VAULT_CLIENT_NAMESPACES}; do
-        echo "Updating configmap in ${n}"
-        kubectl -n "${n}" create configmap "${secret_name}" \
-            --from-file /etc/tls/ca.crt \
-            --dry-run -o yaml | kubectl -n "${n}" replace -f -
+        exit_code="1"
+        until [ "${exit_code}" = "0" ]; do
+            echo "Updating configmap in ${n}"
+            kubectl -n "${n}" create configmap "${secret_name}" \
+                --from-file /etc/tls/ca.crt \
+                --dry-run -o yaml | kubectl -n "${n}" replace -f -
+            exit_code=$(echo $?)
```

Comment:
A small `sleep` here and in the other blocks would prevent spamming `kubectl` continually in the case where there is an issue.